dagster/implnets/workflows/tasks/tasks/assets/all_graph_stats.py

```diff
@@ -55,10 +65,25 @@ def sos_types( ):
     s3Minio = s3.MinioDatastore( GLEANERIO_MINIO_ADDRESS, MINIO_OPTIONS)
     #data = f.getvalue()
 
-
     bucketname, objectname = s3Minio.putReportFile(GLEANER_MINIO_BUCKET,"all","sos_types.csv",report_csv)
     return bucketname, objectname, report_csv
 
+@asset(group_name="graph")
+def all_report_stats():
+    s3Minio = s3.MinioDatastore( GLEANERIO_MINIO_ADDRESS, MINIO_OPTIONS)
+    bucket = GLEANER_MINIO_BUCKET
+    source_url = GLEANERIO_CSV_CONFIG_URL
+
+    community_list = ["all", "deepoceans", "ecoforecast", "geochemistry"]
```

Comment:
put an @TODO to remove hard coding this, later.

Reply:
Sounds good. Added!